package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/angular/vehicle-inspection-ng.js",
   "scripts": {
     "build": "wfm-template-build -m 'wfm.vehicle-inspection'",
-    "watch": "wfm-template-build -w -m 'wfm.vehicle-inspection'"
+    "watch": "wfm-template-build -w -m 'wfm.vehicle-inspection'",
+    "test": "grunt"
```

Comment:
Hi @evanshortiss . Haven't talked in ages!! Thanks for the feedback.

We are introducing grunt to the raincatcher components to get a more accessible build system. The eslint job is just the start, but eventually all of the build tasks will be done through grunt.